src/components/DpTooltip/utils/tooltip.js

```diff
@@ -72,7 +63,7 @@ const initTooltip = (el, value, options) => {
     options,
     zIndex
   )
-  handleHideTooltip = () => hideTooltip(document.getElementById(el.getAttribute('aria-describedby')))
+  handleHideTooltip = () => deleteTooltip(document.getElementById(el.getAttribute('aria-describedby')))
```

Comment:
Nitpick: As we're not actually hiding anything here, i'd suggest to unify wording:

```suggestion
  handleRemoveTooltip = () => removeTooltip(document.getElementById(el.getAttribute('aria-describedby')))
```

Reply:
843c1dee22976879111309c39c5ee7e470acbb14